apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentContainer.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+
+import * as React from 'react';
+import {
+  Touchable,
+  Icon,
+  Text,
+  StyleSheet,
+} from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { Animated } from 'react-native';
+
+type Props = {||};
+
+type State = {|
+  +expandSegments: boolean,
+|};
+
+export default class SegmentContainer extends React.Component<Props, State> {
```

Comment:
Do we have to create `Props` type? It should be possible to add there just `null` instead of `Props`

Reply:
Nope, Props needs to be an object, 
![screenshot 2019-02-20 at 15 33 36](https://user-images.githubusercontent.com/7705041/53098888-ea72d600-3524-11e9-8337-875241fcc380.png)